.gitignore

```diff
@@ -8,3 +8,4 @@ develop-eggs/
 build/
 dist/
 *.egg-info/
+.tox/
```

Comment:
This file is missing a trailing newline.

I don't know if it matters, I know that some tools (git) and people (me) notice and are annoyed by it.

Reply:
newline added